libraries/RainMaker/examples/RMakerCustom/RMakerCustom.ino

```diff
@@ -26,24 +26,67 @@ static Device my_device("Dimmer", "custom.device.dimmer", &gpio_dimmer);
 
 void sysProvEvent(arduino_event_t *sys_event)
 {
-    switch (sys_event->event_id) {
-        case ARDUINO_EVENT_PROV_START:
-#if CONFIG_IDF_TARGET_ESP32S2
-            Serial.printf("\nProvisioning Started with name \"%s\" and PoP \"%s\" on SoftAP\n", service_name, pop);
-            printQR(service_name, pop, "softap");
-#else
-            Serial.printf("\nProvisioning Started with name \"%s\" and PoP \"%s\" on BLE\n", service_name, pop);
-            printQR(service_name, pop, "ble");
-#endif
-            break;
-        case ARDUINO_EVENT_PROV_INIT:
-            wifi_prov_mgr_disable_auto_stop(10000);
-            break;
-        case ARDUINO_EVENT_PROV_CRED_SUCCESS:
-            wifi_prov_mgr_stop_provisioning();
-            break;
-        default:;
-    }
+  switch (sys_event->event_id)
+  {
+    case ARDUINO_EVENT_WIFI_STA_GOT_IP:
+      {
```

Comment:
Please look at the code format in the example and format it accordingly.

Reply:
Update other examples as well.

Reply:
> Please look at the code format in the example and format it accordingly.

Hi!

Could you add a lint with clang-format or similar to Github action to auto format according to Espressif style on PR?

Best regards.